scripts/deploy/govcms-db-sync

```diff
@@ -62,9 +68,33 @@ fi
 echo "[info]: Preparing database sync"
 
 # shellcheck disable=SC2086
-drush --alias-path="$GOVCMS_SITE_ALIAS_PATH" @"$GOVCMS_SITE_ALIAS" sql:dump --gzip --extra-dump=--no-tablespaces --result-file=/tmp/sync.sql -y
+drush --alias-path="$GOVCMS_SITE_ALIAS_PATH" @"$GOVCMS_SITE_ALIAS" sql:dump \
+  --gzip --extra-dump=--no-tablespaces --result-file=/tmp/sync.sql \
+  --skip-tables-key=common -y
 drush rsync --alias-path="$GOVCMS_SITE_ALIAS_PATH" @"$GOVCMS_SITE_ALIAS":/tmp/sync.sql.gz /tmp/ -y
-gunzip < /tmp/sync.sql.gz | drush sqlc
+
+if [[ "$DB_LOAD_NO_DRUSH" = TRUE ]]; then
+  DB_CONF=$(drush sql:conf --show-passwords --format=json 2>&1)
+  DB_NAME=$(echo "$DB_CONF" | jq -r '.database')
+  DB_HOST=$(echo "$DB_CONF" | jq -r '.host')
+  DB_USER=$(echo "$DB_CONF" | jq -r '.username')
+  DB_PASS=$(echo "$DB_CONF" | jq -r '.password')
+
+  LOAD_TMP_FILE=$(mktemp -u --suffix=.govcms-db-load)
+  cat > "$LOAD_TMP_FILE" << EOF
+[client]
+user=${DB_USER}
+password=${DB_PASS}
+EOF
+
+  echo "[info]: Loading database using mysql"
+  gunzip < /tmp/sync.sql.gz | mysql --defaults-file="$LOAD_TMP_FILE" \
+    --host="$DB_HOST" --port=3306 "$DB_NAME"
```

Comment:
Should we pull `port` out here as well just to be sure? Drush should give us the port as well and we'll likely use the standard 3306 port but it _might_ change.

```
{
    "driver": "mysql",
    "database": "drupal",
    "username": "drupal",
    "password": "drupal",
    "port": 3306,
    "charset": "utf8mb4",
    "collation": "utf8mb4_general_ci",
    "host": "mariadb",
    "prefix": "",
    "namespace": "Drupal\\mysql\\Driver\\Database\\mysql",
    "autoload": "core/modules/mysql/src/Driver/Database/mysql/"
}
```